templates/shrine/src/shrine-detail.html

```diff
@@ -243,7 +254,8 @@
           <h1>[[item.title]]</h1>
           <p>[[item.description]]</p>
           <div class="quantitySelector">
-            <select>
+            <label class="sr-only" for="quantity">Number of items:</label>
+            <select id="quantity">
```

Comment:
Agreed, if you have to use an id or class for something like this, there's probably a better way to do it.